mungegithub/mungers/publish_scripts/apimachinery_sync_from_kubernetes.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Was there no reasonable way to get the sync script from the repo?  In openshift we ran two repos for a long time (still do for some things), but it does make it harder to describe where people can find, modify, and test changes.